### PR TITLE
Add scalars download info

### PR DESCRIPTION
**Pull request by ersin-ertan**

### CODE REVIEW

**Comment by JakeWharton** [comment]
This either needs to go in all the converters or none of them. Also I do not want the version number in these files since that would require updating all of them.

**Comment by ersin-ertan** [comment]
Sure, I can do all of them.

How about [this](https://github.com/square/retrofit/pull/2093/commits/46be888e9e2d3ea345a53f7e774e88f7d277e0e4) format?

**Comment by Jawnnypoo** [comment]
Whatever is decided on as far as the versions goes, I'll be sure to apply it here https://github.com/square/retrofit/pull/2082

File: retrofit-converters/scalars/README.md [hunk 1]
```diff
@@ -3,3 +3,25 @@ Java Scalars Converter
...
+```
+or [Gradle][2]:
+```groovy
+compile 'com.squareup.retrofit2:converter-scalars:see.latest.version'
+```
+
+Snapshots of the development version are available in [Sonatype's `snapshots` repository][snap].
+
+ [1]: https://search.maven.org/remote_content?g=com.squareup.retrofit2&a=converter-scalars&v=LATEST
+ [2]: https://mvnrepository.com/artifact/com.squareup.retrofit2/converter-scalars
```
Comment (JakeWharton) [inline retrofit-converters/scalars/README.md]: Please use search.maven.org for this as well. mvnrepository is a third-party knock-off site.

**Comment by JakeWharton** [comment]
This looks fine. Please do it for all of them.

**Comment by Jawnnypoo** [comment]
@ersin-ertan Looks like this is duplicating the work here https://github.com/square/retrofit/pull/2082

**Comment by ersin-ertan** [comment]
It appears so, I had a red bull so forgive my narrow sightedness.

**Comment by ersin-ertan** [comment]
I'm not so sure what should be done for rx java 1 and 2 because there is no rxjava1 and rxjava2 artifacts. It could look like... someone would have to work out the correct artifacts.

RxJava Adapter
==============

An `Adapter` for adapting [RxJava][1] types.

Download
--------

Download [the latest JAR][2] or grab via [Maven][3]:
```xml
<dependency>
  <groupId>com.squareup.retrofit2</groupId>
  <artifactId>adapter-rxjava</artifactId>
  <version>see.latest.version</version>
</dependency>
```
or [Gradle][2]:
```groovy
compile 'com.squareup.retrofit2:adapter-rxjava:see.latest.version'
```

Snapshots of the development version are available in [Sonatype's `snapshots` repository][snap].

 [1]: https://github.com/ReactiveX/RxJava/tree/1.x
 [2]: https://search.maven.org/remote_content?g=com.squareup.retrofit2&a=adapter-rxjava&v=LATEST
 [3]: http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22adapter-rxjava%22
 [snap]: https://oss.sonatype.org/content/repositories/snapshots/

**Comment by JakeWharton** [comment]
Merged manually. Thanks!

**Pull request closed without merging**